Sources/FluentUI_iOS/Components/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -236,12 +238,22 @@ open class BottomCommandingController: UIViewController, TokenizedControl {
     }
 
     /// Initializes the bottom commanding controller with a given content view controller.
-    /// - Parameter contentViewController: View controller that will be displayed below the bottom commanding UI.
-    @objc public init(with contentViewController: UIViewController?) {
+    /// - Parameters:
+    ///  - contentViewController: View controller that will be displayed below the bottom commanding UI.
+    ///  - bottomSheetControllerStyle: The style override for the BottomSheet's background material.
+    @objc public init(with contentViewController: UIViewController?, bottomSheetControllerStyle: BottomSheetControllerStyle) {
```

Comment:
seems minor but since it's a part of our public api, can we rename this to style to match our other controls